src/stylesheets/settings/_settings-components.scss

```diff
@@ -27,6 +27,7 @@ $theme-alert-bar-width: 1 !default;
 $theme-alert-font-family: "ui" !default;
 $theme-alert-icon-size: 5 !default;
 $theme-alert-padding-x: 2.5 !default;
+$theme-alert-padding-y: 2 !default;
 $theme-alert-text-color: default !default;
 $theme-alert-text-reverse-color: default !default;
```

Comment:
Should we deprecate these?

We could use the defaults instead
```
$theme-text-color
$theme-text-reverse-color
```

Reply:
They're maybe a little edge-y but if you wanted a different color for alerts or reverse alerts it would be helpful. I'd prefer not to deprecate them one release after introducing them, I guess. 😬